projects/1.21/assets/rftools-dimensions/rftoolsdim/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+{
+  "block.rftoolsdim.dimlet_workbench": "维度卡工作台",
+  "block.rftoolsdim.knowledge_holder": "知识持有器",
+  "block.rftoolsdim.dimension_builder": "维度构建器",
+  "block.rftoolsdim.dimension_editor": "维度编辑器",
+  "block.rftoolsdim.enscriber": "刻录器",
+  "block.rftoolsdim.block_absorber": "方块吸收器",
+  "block.rftoolsdim.fluid_absorber": "流体吸收器",
+  "block.rftoolsdim.biome_absorber": "生物群系吸收器",
+  "block.rftoolsdim.structure_absorber": "结构吸收器",
+  "block.rftoolsdim.researcher": "研究器",
+  "block.rftoolsdim.activity_probe": "活动探测器",
+
+  "block.rftoolsdim.dimensional_blank_block": "维度方块（空白）",
+  "block.rftoolsdim.dimensional_block": "维度方块",
+  "block.rftoolsdim.dimensional_small_blocks": "维度方块（小型）",
+  "block.rftoolsdim.dimensional_cross_block": "维度方块（交叉）",
+  "block.rftoolsdim.dimensional_cross2_block": "维度方块（交叉2）",
+  "block.rftoolsdim.dimensional_pattern1_block": "维度方块（图案1）",
+  "block.rftoolsdim.dimensional_pattern2_block": "维度方块（图案2）",
+
+  "item.rftoolsdim.empty_dimlet": "空白维度卡",
+  "item.rftoolsdim.attribute_dimlet": "属性维度卡",
+  "item.rftoolsdim.empty_attribute_dimlet": "空白属性维度卡",
+  "item.rftoolsdim.terrain_dimlet": "地形维度卡",
+  "item.rftoolsdim.empty_terrain_dimlet": "空白地形维度卡",
+  "item.rftoolsdim.time_dimlet": "时间维度卡",
+  "item.rftoolsdim.empty_time_dimlet": "空白时间维度卡",
+  "item.rftoolsdim.biome_controller_dimlet": "生物群系控制器维度卡",
+  "item.rftoolsdim.empty_biome_controller_dimlet": "空白生物群系控制器维度卡",
+  "item.rftoolsdim.biome_category_dimlet": "生物群系种类维度卡",
+  "item.rftoolsdim.empty_biome_category_dimlet": "空白生物群系种类维度卡",
+  "item.rftoolsdim.tag_dimlet": "标签维度卡",
+  "item.rftoolsdim.empty_tag_dimlet": "空白标签维度卡",
+  "item.rftoolsdim.biome_dimlet": "生物群系维度卡",
+  "item.rftoolsdim.empty_biome_dimlet": "空白生物群系维度卡",
+  "item.rftoolsdim.feature_dimlet": "特征维度卡",
+  "item.rftoolsdim.empty_feature_dimlet": "空白特征维度卡",
```

```suggestion
  "item.rftoolsdim.feature_dimlet": "地物维度卡",
  "item.rftoolsdim.empty_feature_dimlet": "空白地物维度卡",
```
https://zh.minecraft.wiki/w/%E5%9C%B0%E7%89%A9